integration_test/dummy_helper.erl

```diff
@@ -0,0 +1,75 @@
+-module(dummy_helper).
+
+-required_variable(#{name=>dummy_var, description=>"dummy_var",
```

Comment:
Whitespaces, please :) I'm not going to point every single place where there is a missing space. I hope this is not a big deal to take a look at the diff and pay attention to them. There is also the `lint` plugin for `rebar3` which may be helpful.

Reply:
cannot find any reference to `lint` plugin on rebar3.org I was thinking about elvis integration in the project to "fix" this once and forever :)